### ISSUE TITLE
(commonjs) Function calls don't get rewritten to match rewritten function names

### ISSUE THREAD

**Issue by roubert** [rollup/plugins#606]
- Rollup Plugin Name: commonjs
- Rollup Plugin Version: 15.1.0
- Rollup Version: 2.29.0
- Operating System (or Browser): Linux
- Node Version (if applicable): 10.19.0
- Link to reproduction (IMPORTANT, read below): https://github.com/roubert/rollup-morgan-bug

### Expected Behavior

```
Hello, world!
```

### Actual Behavior

```
rollup-morgan-bug/out/index.js:2126
morgan.format('combined', ':remote-addr - :remote-user [:date[clf]] ":method :url HTTP/:http-version" :status :res[content-length] ":referrer" ":user-agent"');
       ^

TypeError: morgan.format is not a function
```

The problem occurs when _rollup_ processes this source file:

https://github.com/expressjs/morgan/blob/1.10.0/index.js

The calls to `morgan.format()` and `morgan.token()` in there are not being rewritten to match the names that these functions have in the output generated by _rollup_.

Right now I use this _plugin-replace_ hack as a workaround (not pretty, but gets the job done):

https://github.com/roubert/rollup-morgan-bug/tree/workaround

But it seems like something that _rollup_ ought to be able to handle on its own.

**Comment by lukastaegert** [comment]
Done

**Comment by pkit** [comment]
Fixed tests, still fails. No sane errors are given. So much for `clean snapshots` :)

**Comment by lukastaegert** [comment]
There is a good chance I can fix this one as well as I am progressing with #658.

**Comment by pkit** [comment]
@lukastaegert Thanks! Doing circular deps would be a Godsend!

**Comment by lukastaegert** [comment]
I can indeed confirm that this particular issue is already fixed within #658, and without the need to wrap the code. Hope I can get this one ready for release soon.

**Comment by pkit** [comment]
Cool! I will `patch-package` meanwhile.

**Comment by roubert** [comment]
The bot seems to have been a bit overly eager here, this issue is not yet fixed but PR #658 is still in progress.

**Comment by pkit** [comment]
Current workaround is to use `@rollup/plugin-commonjs@beta` channel to install newest plugin.
The OP issue is fixed there.

**Comment by roubert** [comment]
Verified that this was fixed in `@rollup/plugin-commonjs@19.0.0`. This issue can now be closed. Thanks!
